### ISSUE TITLE
package.json not found

### ISSUE THREAD

**Issue by dmaglio** [ricardofbarros/linter-js-standard#17]
```
Error: package.json not found in path
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:13:11)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at findRoot (/home/davide/.atom/packages/linter-js-standard/node_modules/find-root/index.js:20:10)
  at LinterJsStandard.checkStyleSettings (/home/davide/.atom/packages/linter-js-standard/lib/linter-js-standard.coffee:113:19)
  at LinterJsStandard.formatShellCmd (/home/davide/.atom/packages/linter-js-standard/lib/linter-js-standard.coffee:70:8)
  at /home/davide/.atom/packages/linter-js-standard/lib/linter-js-standard.coffee:1:1
  at Config.module.exports.Config.observeKeyPath (/usr/share/atom/resources/app.asar/src/config.js:555:7)
  at Config.module.exports.Config.observe (/usr/share/atom/resources/app.asar/src/config.js:135:21)
  at new LinterJsStandard (/home/davide/.atom/packages/linter-js-standard/lib/linter-js-standard.coffee:44:17)
  at Object.lint (/home/davide/.atom/packages/linter/lib/legacy.coffee:61:22)
  at /home/davide/.atom/packages/linter/lib/editor-linter.coffee:85:24
  at /home/davide/.atom/packages/linter/lib/editor-linter.coffee:84:25
  at Set.forEach (native)
  at EditorLinter.triggerLinters (/home/davide/.atom/packages/linter/lib/editor-linter.coffee:78:26)
  at EditorLinter.lint (/home/davide/.atom/packages/linter/lib/editor-linter.coffee:72:18)
  at /home/davide/.atom/packages/linter/lib/editor-linter.coffee:26:37
  at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at /usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1183:25
```

**Comment by ricardofbarros** [comment]
Nice catch, will fix this as soon as possible.

**Comment by ricardofbarros** [comment]
Fixed on the last release (1.0.6). Thank you :+1:

Please give me some feedback so I can close this issue.
